back/db/migrate/20220531123916_create_home_pages.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class CreateHomePages < ActiveRecord::Migration[6.1]
+  def change
+    create_table :home_pages, id: :uuid do |t|
+      t.boolean :header_avatars_enabled, default: true, null: false
+
+      t.boolean :top_info_section_enabled, default: false, null: false
```

Comment:
Don't we need to add `hero_banner_enabled` and `projects_enabled` (probably) too?

Reply:
Yes, will add!